linux_os/guide/services/apport/service_apport_disabled/rule.yml

```diff
@@ -20,6 +20,8 @@ severity: unknown
 references:
     cis@ubuntu2204: 1.5.3
 
+platform: package[apport]
```

Comment:
You will need to add `apport` to `shared/applicability/package.yml`.

Reply:
I've updated this PR with that change - thank you!